pkg/appliance/backup.go

```diff
@@ -246,6 +246,16 @@ func PerformBackup(cmd *cobra.Command, args []string, opts *BackupOpts) (map[str
 		if err := retryStatus(ctx, b.applianceID, b.backupID); err != nil {
 			return b, err
 		}
+		if opts.Config.Version >= 17 {
```

Comment:
we don't have to worry about version checking here since the new fields are omitted if empty

Reply:
It is also omitted in API version 17<= _if_ the status is not done, which is stated in the spec. Either way, we definitely need some check, either an API version check _or_ a nil check, and since there are more cases where the result field is omitted even on 17<=, a version check is better IMHO.